src/Tools/dotnet-monitor/Commands/ConfigShowCommandHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.IO;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Commands
+{
+    internal sealed class ConfigShowCommandHandler
+    {
+        public static void Invoke(string[] urls, string[] metricUrls, bool metrics, string diagnosticPort, bool noAuth, bool tempApiKey, bool noHttpEgress, ConfigDisplayLevel level)
```

Comment:
Do we need `noHttpEgress` as a parameter? It's no longer being passed to the `Write` method

Reply:
We do if we are telling people "take the parameters from the `collect` command and use them on the `config show` command" to get a replica of the configuration. I can add a comment to that effect.